internal/controller/datadogagent/component/clusteragent/envvar.go

```diff
@@ -10,7 +10,7 @@ const (
 	DDKubeResourcesNamespace         = "DD_KUBE_RESOURCES_NAMESPACE"
 	DDPodName                        = "DD_POD_NAME"
 	DDClusterAgentServiceAccountName = "DD_CLUSTER_AGENT_SERVICE_ACCOUNT_NAME"
-	DDAgentDaemonSet                 = "DD_AGENT_DAEMONSET"
-	DDClusterAgentDeployment         = "DD_CLUSTER_AGENT_DEPLOYMENT"
-	DDDatadogAgentCustomResource     = "DD_DATADOG_AGENT_CUSTOM_RESOURCE"
+	DDAgentDaemonSet                 = "AGENT_DAEMONSET"
```

Comment:
Flare env vars should match the ones in https://github.com/DataDog/datadog-operator/pull/1477